internal/storage/disk/disk.go

```diff
@@ -34,6 +36,7 @@ func init() {
 
 type Store struct {
 	conf *Conf
+	log  *zap.SugaredLogger
```

Comment:
I don't think we need to hold a long term reference to the logger here because it's only used in the `Reload` call. We can just grab a logger instance when the function is called.